charts/nri-bundle/values.yaml

```diff
@@ -2,24 +2,71 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# enables newrelic-infrastructure
 infrastructure:
   enabled: true
 
+# enables nri-prometheus
 prometheus:
   enabled: false
 
+# enables nri-metadata-injection
 webhook:
   enabled: true
 
+# enables kube-state-metrics
 ksm:
   enabled: false
 
+# enables nri-kube-events
 kubeEvents:
   enabled: false
 
+# enables newrelic-logging
 logging:
   enabled: false
 
+# It is possible to configure dependencies values from this chart.
+# For example you can configure verboseLog variable of the newrelic-infrastructure chart installed ad a dependency as follows.
```

Comment:
small typo:  **ad** -> **as**
also maybe add some emphasis on the variable `verboseLog`  to make it easier to see, like bold characters